SConstruct

```diff
@@ -803,6 +814,12 @@ if 'target_sox' in system_dependecies:
 
     tool_env = conf.Finish()
 
+if 'target_libunwind' in system_dependecies:
+    conf = Configure(tool_env, custom_tests=env.CustomTests)
+    if not conf.CheckLibWithHeaderExt('libunwind', 'libunwind.h', 'C', run=not crosscompile):
```

Comment:
```
if 'target_libunwind' in system_dependecies:
    conf = Configure(tool_env, custom_tests=env.CustomTests)
    if not conf.CheckLibWithHeaderExt('libunwind', 'libunwind.h', 'C', run=not crosscompile):
        env.Die("libunwind not found (see 'config.log' for details)")
    tool_env = conf.Finish()
```

A few more issues with this piece:

* we should use `env`, not `tool_env`, because libunwind, unlike sox, is not used only in tools

* the library file is `libunwind.so`, so the compiler option should be `-lunwind`, so the library name here should be `unwind`, not `libunwind`:

      conf.CheckLibWithHeaderExt('unwind', 'libunwind.h', 'C', ...

* we should run pkg-config to gather necessary include and link directories, like: 

      conf = Configure(env, custom_tests=env.CustomTests)
      env.ParsePkgConfig('--cflags --libs libunwind')
      if not conf.CheckLibWithHeaderExt(...